x-conduit/src/X/Data/Conduit.hs

```diff
@@ -0,0 +1,10 @@
+module X.Data.Conduit (
+    yieldM
+  ) where
+
+
+import           Control.Monad.Trans.Class (lift)
+import           Data.Conduit (yield, ConduitM(..))
+
+yieldM :: Monad m => m o -> ConduitM i o m ()
+yieldM mo = lift mo >>= yield
```

Comment:
So conduit exposes another `yieldM` function of `yieldM :: Monad m => m o -> Pipe l i o u m ()` which uses the `Pipe` type and doesn't work seamlessly with the other functions exported by `Data.Conduit` which all expect `ConduitM` or one of it's aliases.